src/page/Variant.tsx

```diff
@@ -398,12 +403,24 @@ class Variant extends React.Component<IVariantProps> {
                         <Col>
                             <FunctionalGroups
                                 myVariantInfo={this.myVariantInfo}
-                                annotationInternal={this.annotationSummary}
+                                annotationInternal={
+                                    this.props.store.annotationSummary
+                                }
                                 variantAnnotation={this.variantAnnotation}
                                 oncokb={this.oncokb}
+                                isCanonicalTranscriptSelected={
+                                    this.isCanonicalTranscriptSelected!
+                                }
                             />
                         </Col>
                     </Row>
+                    {!this.isCanonicalTranscriptSelected && (
+                        <div>
+                            * For the following resources, the transcript is
+                            based on different annotation, but the genomic
+                            change is the same.
```

Comment:
The text says `following resources` but there is nothing following this sentence :) I think we should replace `following` with some other word.

Reply:
@onursumer how about remove `For the following resources,`, just remain `* The transcript is based on different annotation, but the genomic change is the same.`
@inodb do you have any suggestions on the footnote message?

Reply:
`For the following resources, the transcript is based on different annotation, but the genomic change is the same.`

How about

`* This resource's annotation is based on a different transcript than the displayed one, but the genomic change is the same.`